processor/resourcedetectionprocessor/internal/aws/ecs/ecs.go

```diff
@@ -69,6 +69,7 @@ func (d *Detector) Detect(context.Context) (pdata.Resource, error) {
 	attr.InsertString(conventions.AttributeCloudInfrastructureService, conventions.AttributeCloudProviderAWSECS)
 	attr.InsertString("aws.ecs.task.arn", tmdeResp.TaskARN)
 	attr.InsertString("aws.ecs.task.family", tmdeResp.Family)
+	attr.InsertString("aws.ecs.task.revision", tmdeResp.Revision)
```

Comment:
Please define these in go.opentelemetry.io/collector/translator/conventions

Reply:
open-telemetry/opentelemetry-collector#2816, waiting on the spec PR to mark as ready